otx/algorithms/common/adapters/mmcv/models/backbones/torchvision_backbones.py

```diff
@@ -0,0 +1,241 @@
+"""Model templates builder."""
```

Comment:
You need more detail description for this file.
I can't understand the purpose of this file through this description

Reply:
Yes, I will supplement the description of this and write it. Thanks for the advice.

Reply:
Added additional description to main file. Could you please check it?